samples/bot-conversation-sso-quickstart/csharp_dotnetcore/BotConversationSsoQuickstart/Resources/options.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "type": "AdaptiveCard",
+  "body": [
+    {
+      "type": "TextBlock",
+      "text": "",
+      "size": "Medium",
+      "weight": "Bolder"
+    },
+    {
+      "type": "ActionSet",
+      "actions": [
+        {
+          "type": "Action.Submit",
+          "title": "Sign in Universal SSO",
+          "data": {
+            "msteams": {
+              "type": "messageBack",
+              "displayText": "Sign in Universal SSO",
+              "text": "ABSSORefresh",
+              "value": "{\"bfKey\": \"bfVal\", \"conflictKey\": \"from value\"}"
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+  "appId": "9b6dcf71-df47-4756-b486-47ca94dbacee",
+  "version": "1.4"
```

Comment:
Check if this version is correct and supported

Also, why did we push app id ?

Add place holder and Update Readme file to

Reply:
Changes updated